src/main/java/ProgramB.java

```diff
@@ -0,0 +1,57 @@
+import java.text.DecimalFormat;
+import java.util.Scanner;
+/**
+ * 1. 기초 알고리즘 b. 두 개의 숫자를 입력받고, n~m 까지의 연속된 수의 합을 구하는 프로그램
+ */
+public class ProgramB {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        String inputN;
+        String inputM;
+        Verification vrfc = new Verification();
+
+        do {
+            System.out.println("정수 값을 입력해주세요.");
+            inputN = scan.next();
+        } while (vrfc.isString(inputN));
+        do {
+            System.out.println("정수 값을 입력해주세요.");
+            inputM = scan.next();
+        } while (vrfc.isString(inputM));
+
+        long n, m;
+        // 입력받은 수 중 작은 수를 n, 큰 수를 m으로
+        if (Long.parseLong(inputN) <= Long.parseLong(inputM)) {
+            n = Long.parseLong(inputN);
+            m = Long.parseLong(inputM);
+        } else {
+            n = Long.parseLong(inputM);
+            m = Long.parseLong(inputN);
+        }
```

Comment:
```suggestion
       long n = Math.min(inputN, inputM);
       long m = Math.max(inputN, inputM);
```
Math.min, Math.max 를 사용하면 조금 더 쉽게 로직을 만들 수 있어요.

Reply:
지금 한다면 Math 클래스를 썼을텐데 이 때는 그런 클래스도 모르고 구현에 급급해서 이런 코드를 썼었네요..